tests/phpunit/api/v3/DashboardTest.php

```diff
@@ -59,6 +59,24 @@ public function testDashboardCreate() {
     $newCount = CRM_Core_DAO::singleValueQuery('select count(*) from civicrm_dashboard');
     $this->assertEquals($oldCount + 1, $newCount);
     $this->DashboardDelete($dashboard['id'], $oldCount);
+    $this->assertEquals($dashboard['values'][$dashboard['id']]['is_active'], 1);
+  }
+
+  /**
+   * CRM-19217.
+   * Ensure that where is_active is specifically set to 0 is_active returns 0.
+   */
+  public function testDashboardCreateNotActive() {
+    $params = array(
+      'version' => 3,
```

Comment:
If you are updating can you update the comment block - the full standard is there should be a line break after the first line